packages/react-native/third-party-podspecs/glog.podspec

```diff
@@ -29,10 +29,23 @@ Pod::Spec.new do |spec|
                         'src/base/*.h'
   spec.exclude_files       = "src/windows/**/*"
   spec.compiler_flags      = '-Wno-shorten-64-to-32'
+
+  # TODO: T167482718 Remove this code after April 2024, when Apple will
+  # push the lower version of Xcode required to upload apps to the Store.
+  xcode_path = `xcode-select -p` # This return the current version of Xcode
```

Comment:
This makes the assumption that Xcode was installed as `/Applications/Xcode-15.0.0.app/Contents/Developer` etc. It's common for this command to return `/Applications/Xcode.app/Contents/Developer` without a version number.

Reply:
Maybe try `xcodebuild -version` instead?

```
% xcodebuild -version
Xcode 15.0
Build version 15A240d
```

Reply:
Well spotted, I updated the PR using `xcodebuild -version` which is the same logic we used for the Xcode 15 flag workaround and it should be more robust.